config/index.conf.php

```diff
@@ -81,9 +81,6 @@
         'index' => 'items',
         'body' => [
             'mappings' => [
-                '_source' => [
-                    'enabled' => false
-                ],
```

Comment:
Why are we changing this?

Reply:
Because we need the field _source stored on the elastic search to reindex it again using just one query. Otherwise, we can not do it :).